src/components/Navigation.js

```diff
@@ -1,15 +1,17 @@
-import React from 'react';
-import { NavLink } from 'react-router-dom';
+import React, { useContext } from "react";
+import { NavLink } from "react-router-dom";
+import CartContext from "../contexts/CartContext";
 
-const Navigation = props => {
-	return (
-		<div className="navigation">
-			<NavLink to="/">Products</NavLink>
-			<NavLink to="/cart">
-				Cart <span>{props.cart.length}</span>
-			</NavLink>
-		</div>
-	);
+const Navigation = () => {
+  const cart = useContext(CartContext);
+  return (
+    <div className="navigation">
+      <NavLink to="/">Products</NavLink>
+      <NavLink to="/cart">
+        Cart <span>{cart.length}</span>
```

Comment:
CartContext is supposed to return an object, you should destructure it to get cart